assets/cubyz/recipes/other_recipes

```diff
@@ -0,0 +1,4 @@
+cubyz:stone
+cubyz:chisel
+result cubyz:stone_bricks
+result cubyz:chisel
```

Comment:
Please make an issue with a proposal of how this is supposed to work. After all there is only one slot for the players mouse.